From: "Laszlo Boszormenyi (GCS)" <gcs@debian.hu>
Date: Sun, 16 Jun 2013 16:27:23 -0700
Subject: force-reload

---
 etc/init/couchdb.tpl.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/etc/init/couchdb.tpl.in b/etc/init/couchdb.tpl.in
index 39b6250..63aa4a5 100644
--- a/etc/init/couchdb.tpl.in
+++ b/etc/init/couchdb.tpl.in
@@ -132,7 +132,7 @@ parse_script_option_list () {
                 log_end_msg $SCRIPT_ERROR
             fi
             ;;
-        restart)
+        restart|force-reload)
             log_daemon_msg "Restarting $DESCRIPTION" $NAME
             if stop_couchdb; then
                 if start_couchdb; then
@@ -149,7 +149,7 @@ parse_script_option_list () {
             ;;
         *)
             cat << EOF >&2
-Usage: $SCRIPT_NAME {start|stop|restart|status}
+Usage: $SCRIPT_NAME {start|stop|restart|force-reload|status}
 EOF
             exit $SCRIPT_ERROR
             ;;
