From: "Laszlo Boszormenyi (GCS)" <gcs@debian.hu>
Date: Sun, 16 Jun 2013 16:27:23 -0700
Subject: couchdb_sighup

---
 bin/couchdb.tpl.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/bin/couchdb.tpl.in b/bin/couchdb.tpl.in
index 6223b76..d6f26f7 100644
--- a/bin/couchdb.tpl.in
+++ b/bin/couchdb.tpl.in
@@ -273,6 +273,9 @@ stop_couchdb () {
     PID=`_get_pid`
     STOP_TIMEOUT=60
     if test -n "$PID"; then
+        if test "$1" = "false"; then
+            echo > $PID_FILE
+        fi
         if kill -0 $PID 2> /dev/null; then
             if kill -TERM $PID 2> /dev/null; then
                 count=0
