From: "Laszlo Boszormenyi (GCS)" <gcs@debian.hu>
Date: Sun, 16 Jun 2013 16:27:23 -0700
Subject: wait_for_couchdb_stop

---
 etc/init/couchdb.tpl.in | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/etc/init/couchdb.tpl.in b/etc/init/couchdb.tpl.in
index 63aa4a5..8e9b95e 100644
--- a/etc/init/couchdb.tpl.in
+++ b/etc/init/couchdb.tpl.in
@@ -104,6 +104,17 @@ stop_couchdb () {
     # Stop the running Apache CouchDB process.
 
     run_command "$COUCHDB -d" > /dev/null
+    RET=1;
+    for i in $(seq 1 30); do
+        status=`$COUCHDB -s 2>/dev/null | grep -c process`;
+        if [ "$status" -eq 0 ]; then
+            RET=0;
+            break;
+        fi;
+        echo -n .;
+        sleep 1s;
+    done;
+    return $RET
 }
 
 display_status () {
